src/pymovements/gaze/transforms.py

```diff
@@ -711,9 +711,16 @@ def savitzky_golay(
         cval=constant_value,
     )
 
+    # Wrap function to ignore unexpected keyword arguments possibly passed by Polars
```

Comment:
I'm a bit confused on the issue with these unexpected keyword arguments. Are they passed by polars or by pymovements?